README.md

```diff
@@ -26,8 +26,8 @@ The following table lists all models that are materialized within this package b
 | **Model**                | **Description**                                                                                                                                |
 | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
 | [amazon_ads__account_report](https://fivetran.github.io/dbt_amazon_ads/#!/model/model.amazon_ads.amazon_ads__account_report)             | Each record in this table represents the daily performance at the account level. |
-| [amazon_ads__campaign_report](https://fivetran.github.io/dbt_amazon_ads/#!/model/model.amazon_ads.amazon_ads__campaign_report)            | Each record in this table represents the daily performance at the campaign level. |
-| [amazon_ads__portfolio_report](https://fivetran.github.io/dbt_amazon_ads/#!/model/model.amazon_ads.amazon_ads__portfolio_report)            | Each record in this table represents the daily performance at the portfolio level. |
+| [amazon_ads__campaign_report](https://fivetran.github.io/dbt_amazon_ads/#!/model/model.amazon_ads.amazon_ads__campaign_report)            | Each record in this table represents the daily performance at the campaign level. | 
+| [amazon_ads__portfolio_report](https://fivetran.github.io/dbt_amazon_ads/#!/model/model.amazon_ads.amazon_ads__portfolio_report)            | Each record in this table represents the daily performance at the portfolio level. | 
```

Comment:
It looks like we stuck to the hierarchy when documenting the end models. Since portfolio is a higher grain than campaign (correct me if I am wrong) then can we switch the two in the table view.

Reply:
done.